client/my-sites/beta-testing/main.jsx

```diff
@@ -0,0 +1,194 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+import FollowButtonContainer from 'calypso/blocks/follow-button';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'calypso/components/main';
+import { Button, Card } from '@automattic/components';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { getSelectedSite, getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import DocumentHead from 'calypso/components/data/document-head';
+import QuerySiteChecklist from 'calypso/components/data/query-site-checklist';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	getPostsForQueryIgnoringPage,
+	isRequestingPostsForQueryIgnoringPage,
+} from 'calypso/state/posts/selectors';
+import QueryPosts from 'calypso/components/data/query-posts';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import CardHeading from 'calypso/components/card-heading';
+
+const query = {
+	author: null,
+	number: 3,
+	order: 'DESC',
+	status: 'publish',
+	type: 'post',
+	category: 'testing',
+};
+const horizonSiteId = 90972941;
+
+const BetaTesting = ( { siteId, posts, isRequestingPosts, trackViewHorizonAction } ) => {
+	const translate = useTranslate();
+	const header = (
+		<FormattedHeader
+			brandFont
+			headerText={ translate( 'Beta Testing', { context: 'Plugin Installer'} ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **122** times:
`translate( 'Beta Testing', { context: 'Plugin Installer'} )` *ES Score: 16*